src/charts/raster-chart.js

```diff
@@ -605,7 +605,7 @@ export default function rasterChart(parent, useMap, chartGroup, _mapboxgl) {
       .then(results => callback(results[0]))
       .catch(error => {
         throw new Error(
-          `getResultRowForPixel failed with message: ${error.message}`
+          `getResultRowForPixel failed with message: ${error.error_msg}`
```

Comment:
What's the reason for changing from the standard JS [Error.prototype.message](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/message)? 

Not sure if it's related to this error since it's reportedly reproduced on master: https://github.com/omnisci/mapd-immerse/pull/7301#issuecomment-736739723

Reply:
yeah, this is not related with this PR. Tamara's error is not related with this. The error object doesn't have `message` property but `error_msg`. I am not sure when the response changed, but because of this we are not seeing the error message in the console.

Reply:
The error she sees is the error that comes up when hit testing call fails. With this ^ change, we can see what the actual error message is.

Reply:
👍